docs/dev_guide/deprecations/chado_cv_api.rst

```diff
@@ -0,0 +1,30 @@
+
+Deprecations in tripal_chado.variables.api.inc
```

Comment:
It looks like this file is still a copy of the variables file, so it needs to be updated